libr/bin/format/elf/elf.c

```diff
@@ -3351,6 +3361,59 @@ ut64 Elf_(r_bin_elf_v2p) (ELFOBJ *bin, ut64 vaddr) {
 	return vaddr;
 }
 
+/* converts a physical address to the virtual address, looking
+ * at the program headers in the binary bin */
+ut64 Elf_(r_bin_elf_p2v_new) (ELFOBJ *bin, ut64 paddr) {
```

Comment:
there is code that depends on the old behavior of the function ... i could fix them now but that can result in one huge PR that I would rather avoid

Reply:
i plan to rename `r_bin_elf_p2v_new` back to `r_bin_elf_p2v` once i've fixed all the users of the latter ... like you said, this isn't a public API so no need to worry about outside code

Reply:
It would be big but simple, because the change is always the same. I'd avoid half-done PRs, because they end up remaining half-done for a long time in general.

Reply:
maybe but that doesn't mean the code won't work in the meantime ... besides, this pr was initiated by me, doesn't seem particularly difficult to complete to everyone's satisfaction (but will take some time) and you can just remind me on Telegram if it seems dormant

Reply:
ok but if we all add "improved" functions without fixing/removing the old one it will soon become a mess. It's not only about working code, code should be maintained as well.

Reply:
agree

Reply:
so i'm not very clear on this ... better merge it in now or wait until it is complete?

Reply:
let's complete it another time.

Reply:
ok thanks!